Makefile.include

```diff
@@ -327,12 +327,12 @@ APPLICATION := $(strip $(APPLICATION))
 
 ifeq (,$(and $(DOWNLOAD_TO_STDOUT),$(DOWNLOAD_TO_FILE)))
   ifeq (,$(WGET))
-    ifeq (0,$(shell which wget 2>&1 > /dev/null ; echo $$?))
+    ifeq (0,$(shell which wget > /dev/null 2>&1 ; echo $$?))
```

Comment:
Is it even nececary to redirect stderr to stdout at this point?

Reply:
I was trying for minimal changes here; as that's apparently not called for (cf. https://github.com/RIOT-OS/RIOT/pull/16775#issuecomment-904639562), the latest additions do away with this.